src/BuildPredictionTests/Predictors/ServiceFabricServiceManifestPredictorTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Build.Prediction.Tests.Predictors
+{
+    using System.IO;
+    using Microsoft.Build.Construction;
+    using Microsoft.Build.Evaluation;
+    using Microsoft.Build.Prediction.Predictors;
+    using Xunit;
+
+    public class ServiceFabricServiceManifestPredictorTests
+    {
+        [Fact]
+        public void FindItems()
+        {
+            Project project = CreateTestProject("project.sfproj");
+            var expectedInputFiles = new[]
+            {
+                new PredictedItem($@"Service1\PackageRoot\{ServiceFabricServiceManifestPredictor.ServiceManifestFileName}", nameof(ServiceFabricServiceManifestPredictor)),
+                new PredictedItem($@"Service2\PackageRoot\{ServiceFabricServiceManifestPredictor.ServiceManifestFileName}", nameof(ServiceFabricServiceManifestPredictor)),
+                new PredictedItem($@"ServiceFabricApp\ApplicationPackageRoot\Foo\{ServiceFabricServiceManifestPredictor.ServiceManifestFileName}", nameof(ServiceFabricServiceManifestPredictor)),
+                new PredictedItem($@"ServiceFabricApp\ApplicationPackageRoot\Bar\{ServiceFabricServiceManifestPredictor.ServiceManifestFileName}", nameof(ServiceFabricServiceManifestPredictor)),
+            };
+            new ServiceFabricServiceManifestPredictor()
+                .GetProjectPredictions(project)
+                .AssertPredictions(
+                    project,
+                    expectedInputFiles.MakeAbsolute(Directory.GetCurrentDirectory()),
```

Comment:
Tests might need a slight refactoring. The tests which have test data generally make expectations relative to their project. So far this has been the case for predictor tests which construct their projects dynamically, but now that there are dependencies involved there seems to be a need for a root directory for the test (probably a temp one too to avoid tainting other tests). This is out of scope for this change however.